dockers/retirejs/Dockerfile

```diff
@@ -7,6 +7,7 @@ RUN apk update && apk upgrade \
 	&& apk add --no-cache alpine-sdk bash openssh-client \
 	&& apk add git
 
+RUN npm install
 RUN npm install -g retire
 
 RUN wget -O jq https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
```

Comment:
By checking your containers' command in `config.yml` file, I realized that you are not actually using `jq` during your securityTest. What you think about removing  these `jq` lines? 😃

Reply:
Sure can! Fixed the unnecessary command in 1dc1c00